qemu/qemu_test.go

```diff
@@ -1,4 +1,5 @@
 /*
+// Copyright (c) 2018 Yash Jain
```

Comment:
I'm not really the best person to comment on this but you haven't actually added any code to this file.  You've just deleted stuff.  Hence I find the change to the copyright a little confusing.  Anyone else care to comment?